package.json

```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/Mor-s-organization/misconfiguration-check#readme",
   "dependencies": {
-    "lodash": "^4.17.20"
+    "axios": "^0.19.2"
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-0536   
**Check Name:** follow-redirects: Exposure of Sensitive Information via Authorization Header leak   
**Severity:** MEDIUM   
**Fixed Version:** 1.14.8   
**Description:** Exposure of Sensitive Information to an Unauthorized Actor in NPM follow-redirects prior to 1.14.8.
(This package is used under: axios->follow-redirects)  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-0536